Library/Homebrew/cmd/test-bot.rb

```diff
@@ -425,6 +426,15 @@ def formula formula_name
       install_args = %w[--verbose]
       install_args << "--build-bottle" unless ARGV.include? "--no-bottle"
       install_args << "--HEAD" if ARGV.include? "--HEAD"
+
+      # Pass --devel or --HEAD to the test-bot in the event formulae lack stable. Supports devel-only/head-only.
+      # head-only should not have devel, but devel-only can have head. Stable can have all three.
+      if devel_only?(formula)
```

Comment:
Just FYI on my personal style in here: I hate parens and only use them when things break without :)